.github/workflows/tag-staging.yml

```diff
@@ -17,12 +20,32 @@ jobs:
           for number in $(gh pr list --label "staging:active" --json number | jq -r '.[].number'); do
           gh pr edit ${number} --remove-label "staging:active"
           done
-      - name: Push to staging branch
-        run: |
-          git checkout -b staging-pr-$PRNUM
-          git push -f origin staging-pr-$PRNUM:staging
+      - name: Build
+        run: docker build -t foo .
+      - name: Push to ECR
+        id: ecr
+        uses: jwalton/gh-ecr-push@v1
+        with:
+          access-key-id: ${{ secrets.ECR_AWS_ACCESS_KEY_ID }}
+          secret-access-key: ${{ secrets.ECR_AWS_SECRET_ACCESS_KEY }}
+          region: eu-west-2
+          local-image: foo
```

Comment:
Is foo a temporary name or is it worth being more descriptive?

Reply:
It's just a temporary name. Copied straight out of Cloud Platform's scripts 😇